packages/api/src/index.ts

```diff
@@ -38,7 +38,15 @@ void (async () => {
           await entityManager.persistAndFlush(newUser);
         }
 
-        done(null, { profile, accessToken, refreshToken });
+        const guildIds = profile.guilds?.filter((guild) => guild.id === env.discordGuildId) ?? [];
```

Comment:
I'd prefer something like this only because `guildIds` implies that it's ALL of the guild IDs and it makes line 43 ambiguous (it currently reads like `if the user is in a guild, they're good to go`.

@johnkahn thoughts? Is there a better way to do the `map` business?

```suggestion
        const isMemberOfEventGuild = (profile.guilds ?? []).map((guild) => guild.id).includes(env.discordGuildId);
```

Reply:
Nope, looks good to me